bundles/org.openhab.binding.doorbird/src/main/java/org/openhab/binding/doorbird/internal/api/DoorbirdAPI.java

```diff
@@ -102,6 +102,21 @@ public void setHttpClient(HttpClient httpClient) {
         return doorbirdInfo;
     }
 
+    public @Nullable DoorbirdSession getSession() {
+        DoorbirdSession doorbirdSession = null;
+        try {
+            String sessionResponse = executeGetRequest("/bha-api/getsession.cgi");
+            doorbirdSession = new DoorbirdSession(sessionResponse);
+        } catch (IOException e) {
+            logger.info("Unable to communicate with Doorbird: {}", e.getMessage());
```

Comment:
```suggestion
            logger.warn("Unable to communicate with Doorbird: {}", e.getMessage());
```
Don't you also want to set the thing status to offline communication eror? If so, it can be set to debug, as it provides some more detail then the thing status

Reply:
I'll change this and similar methods to to use `warn` instead of `info`.

As for setting the thing OFFLINE, I'd prefer to hold off on that as it requires changes in a number of other places to make sure it is set back ONLINE properly. And, since the device is on the users local network, the likelihood of these errors is pretty low.

WDYT?

Reply:
A warn message may allready be a good step forward for users to troubleshoot connectivity issues.
That being said, i would recommend to have a proper thing status handling. I also understand that this PR's scope is adding a V2 API and not improving the allready existing thangstatus handling. So maybe we should leave that to a future PR.

Reply:
Ok, thanks.